Sources/SwiftFormat/Rules/OmitReturns.swift

```diff
@@ -0,0 +1,157 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+/// Single-expression functions, closures, subscripts can omit `return` statement.
+///
+/// Lint: `func <name>() { return ... }` and similar single expression constructs will yield a lint error.
+///
+/// Format: `func <name>() { return ... }` constructs will be replaced with
+///         equivalent `func <name>() { ... }` constructs.
+@_spi(Rules)
+public final class OmitReturns: SyntaxFormatRule {
```

Comment:
Can we use `OmitSingleExpressionReturns` as the rule name instead, so it's a little more self-descriptive?

Reply:
This is more confusing to me, maybe `OmitExplicitReturns`? Verbose version is `OmitExplicitReturnsFromSingleExpressionBodies`